docs/components/label.md

```diff
@@ -0,0 +1,42 @@
+# Label
+
+**KLabel** provides a wrapper around general `label` tags
+
+## Standard Input
+
+<br />
+
+<KLabel>Input Title</KLabel>
+
+```vue
+<KLabel>Input Title</KLabel>
+```
+
+## With Help
+
+<br />
+
+<KLabel help="This is an example">Input Title</KLabel>
+
+```vue
+<KLabel help="This is an example">Input Title</KLabel>
+```
+
+<br />
+
+<KLabel help="This is a really long tooltip. Hopefully we won't have anything this long but we might. I wonder how it handles long inputs">Long Input Title</KLabel>
+
+```vue
+<KLabel
+  help="This is a really long tooltip. Hopefully we won't have anything this long but we might. I wonder how it handles long inputs"
+>
+  Long Input Title
+</KLabel>
+```
+
+## Sample with an input + tooltip
+
+<br />
+
+<KLabel tooltip="A service is an API that you want to offer">Service Name</KLabel>
```

Comment:
When I used `help` locally and hovered over the icon for +2 seconds, I saw 2 tooltips - one being the intended tooltip and the other "help" coming from somewhere else. I think we need to find and suppress the unintended tooltip:

![Screen Shot 2021-06-02 at 8 03 51 PM](https://user-images.githubusercontent.com/2080476/120580537-38523600-c3de-11eb-922a-8c48b0c82898.png)

Reply:
Good catch! This was related to the `<title>` property in `KIcon`. I've added an attribute to disable it (default to `false` to keep backwards compatibility)